src/clients/evm/index.ts

```diff
@@ -52,8 +57,18 @@ export class SnapshotEVMClient {
       quorum,
       votingStrategies,
       authenticators,
-      executionStrategies
+      executionStrategies,
+      salt
     );
+
+    const receipt = await response.wait();
```

Comment:
Shouldnt this happen on the UI? I assume doing this you wont be able to get the tx hash and show it on the pending transaction section in the UI.